python/lsst/ctrl/mpexec/cli/script/qgraph.py

```diff
@@ -135,6 +137,9 @@ def qgraph(  # type: ignore
         QuantumGraph.
     clobber_execution_butler : `bool`
         It True overwrite existing execution butler files if present.
+    transfer : `str`
+        Transfer mode for execution butler creation. The default option,
+        "none", means that no files will be transferred.
```

Comment:
Where is `none` coming in as an option? It's not the available in the standard list.  `auto` will not quite work if the source and destination are both POSIX because it will try symlink before it tries copy if the target and source roots differ. That might be fine though since you are likely going to want an explicit "copy" here with a new root. "direct" would leave all the existing files where they are but let you write new files to the new root.

You could make "auto" the default if no root is specified and "copy" the default if a new root is given.